live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/in/mapper/property/PropertiesMapperTest.java

```diff
@@ -144,12 +147,64 @@ void shouldHandleAllCustomPropertiesUpdated_NodeCreated()
 
         // then
         Set<CustomPropertyDelta<?>> expectedPropertyDeltas = Set.of(
+                CustomPropertyDelta.updated("cm:name", "some name"),
                 CustomPropertyDelta.updated("cm:title", "some title"),
                 CustomPropertyDelta.updated("cm:description", "some description"));
 
         assertEquals(expectedPropertyDeltas, customPropertyDeltas);
     }
 
+    @Test
+    void shouldPutNameInCustomProperties_NodeCreated()
+    {
+        // given
+        String name = "some name";
+
+        RepoEvent<DataAttributes<NodeResource>> event = mock();
+
+        setType(event, NODE_CREATED);
+
+        NodeResource nodeResource = NodeResource.builder()
+                .setName(name)
+                .setProperties(Map.of())
```

Comment:
Do we need this line?

Reply:
hmm, it throws npe if properties are not set but maybe actually I should fix the code instead of setting it to empty map here🤔

Reply:
fixed